mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6132 applyArmorToDamage (Lnet/minecraft/class_1282;F)F
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_23883 dropXp ()V
+	METHOD method_23733 onKilledBy (Lnet/minecraft/class_1309;)V
+		COMMENT Performs secondary effects after this mob has been killed.
+		COMMENT 
+		COMMENT <p> The default behaviour spawns a wither rose if {@code adversary} is a {@code WitherEntity}.
```

Comment:
Missing closing tag?

Reply:
Javadoc never closes <p> tags for some reason, check official docs

Reply:
Yeah, html auto closes `<p>` tags when it meets another `<p>` tag